### ISSUE TITLE
Unidentified Artifact recipes

### ISSUE THREAD

**Issue by QwertRvg** [Pandarix/BetterArcheology#71]
Hello! I am using your mod in my modpack, but find the Unidentified Artifact to be confusing for the player.
Could you add recipes to JEI/EMI for the Unidentified Artifact. 
There could simply be recipes for all items that can come from the Unidentied Artifact.

So something like the image from your wiki:
![image](https://github.com/Pandarix/BetterArcheology/assets/89683173/ba349f99-30e2-40a6-9ecb-7f2996b8008a)

But for every possible item. That way players knew both how to use that item and how to get all the items that can come from it.

**Comment by Pandarix** [comment]
I got stuck once with this issue because the recipe needs to work for enchanted books and the nbt tomfoolery confused me. Now that I am fixing other bugs, I stumbled across this issue again and found a rather good working way to do this.

TLDR: This will be included in the next update. Sorry for the wait.

**Comment by Pandarix** [comment]
## Fix/ Feature is now out in BetterArcheology 1.2.0!

This issue will be closed. If the problem still persists or similar, feel free to reopen this issue.

## Links
[![Modrinth Downloads](https://img.shields.io/modrinth/dt/better-archeology?style=flat&logo=modrinth&link=https%3A%2F%2Fmodrinth.com%2Fmod%2Fbetter-archeology%2Fversions)](https://modrinth.com/mod/better-archeology/versions) [![CurseForge Downloads](https://img.shields.io/curseforge/dt/835687?style=flat&logo=curseforge&logoColor=orange&color=orange)
](https://www.curseforge.com/minecraft/mc-mods/better-archeology/files)
Please wait until CurseForge has verified the files and wait for the 1.2.0 :)

**Comment by QwertRvg** [comment]
thanks so much!
